src/native/libs/System.Native/pal_time.c

```diff
@@ -95,6 +96,29 @@ uint64_t SystemNative_GetTimestamp()
 #endif
 }
 
+int64_t SystemNative_GetBootTimeTicks()
+{
+#ifdef TARGET_LINUX
```

Comment:
It appears that `TARGET_LINUX` isn't set for Android and so we need to check for `TARGET_ANDROID` if we want to use it on Android as well.

```suggestion
#if defined(TARGET_LINUX) || defined(TARGET_ANDROID)
```

Reply:
Ah, you are right. I thought `TARGET_LINUX` is set for Android which is not the case. In cmake context, `CLR_CMAKE_HOST_LINUX` is set for Android:

https://github.com/dotnet/runtime/blob/283e351d0406d455a035b651b3d96b0209e73450/eng/native/configureplatform.cmake#L146-L148

but then we use `elseif` to skip `TARGET_LINUX` on Android:
https://github.com/dotnet/runtime/blob/283e351d0406d455a035b651b3d96b0209e73450/eng/native/configurecompiler.cmake#L529-L532

Nice catch!